DEPLOY_VERIFIER.md

```diff
@@ -0,0 +1,77 @@
+
+        // The salt used (can be any bytes32 value)
+        bytes32 salt = bytes32(0); // Using zero salt as per your requirements
```

Comment:
probs this comment can be changed
`Using zero salt as per your requirements`

Reply:
Hah, nice catch, ChatGPT leftover